tests/unit/test_dao.py

```diff
@@ -157,20 +158,16 @@ async def test_db_timeout_error_translator():
     not_timeout_error = PyMongoError()  # .timeout is False by default
 
     # Non-timeout errors should be re-raised as they are
```

Comment:
```suggestion
    # Non-timeout errors should be translated to base DaoErrors
```

Reply:
See [5daea3c](https://github.com/ghga-de/hexkit/pull/133/commits/5daea3c63eba2fda08ed7925a13e10ffa9883550) for a fix